helix-term/src/commands/lsp.rs

```diff
@@ -49,7 +49,7 @@ use std::{
 /// If there is no configured language server that supports the feature, this displays a status message.
 /// Using this macro in a context where the editor automatically queries the LSP
 /// (instead of when the user explicitly does so via a keybind like `gd`)
-/// will spam the "No configured language server supports <feature>" status message confusingly.
+/// will spam the "No configured language server supports \<feature>" status message confusingly.
```

Comment:
I couldn't decide whether to backquote `<feature>` (so it appears as "code") or just escape the HTML tag rustdoc was complaining about. Happy to go either way.

Reply:
Escaping here seems proper to me 👍